projects/igniteui-angular/src/lib/avatar/avatar.component.ts

```diff
@@ -111,18 +263,32 @@ export class IgxAvatarComponent implements OnInit, AfterViewInit {
         }
 
         return this.iconTemplate;
+
     }
 
     constructor(public elementRef: ElementRef) { }
 
+    /**
+     * @hidden
+     *
+     * @memberof IgxAvatarComponent
+     */
     public ngOnInit() {
         this.roleDescription = this.getRole();
     }
 
+    /**
+     *@hidden
+     *
+     * @memberof IgxAvatarComponent
```

Comment:
* @memberof IgxAvatarComponent - remove it, if the property/method/input is hidden.